cloudflare.loader.php

```diff
@@ -23,6 +23,12 @@
 // Initiliaze Hooks class which contains WordPress hook functions
 $cloudflareHooks = new \CF\WordPress\Hooks();
 
+/*
+ * php://input can only be read once before PHP 5.6, try to grab it IF the request
+ * is coming from the cloudflare proxy
+ */
+add_action('plugins_loaded', array($cloudflareHooks, 'getCloudflareRequestJSON'));
```

Comment:
Let's be more clear by changing `try to grab it IF the request` to `try to grab it ONLY if the request`.